testing/idris/APKBUILD

```diff
@@ -0,0 +1,44 @@
+#-*-mode: Shell-script; coding: utf-8;-*-  
+# Maintainer: Mitch Tishmack <mitch.tishmack@gmail.com>
+pkgname=idris
+pkgver=1.0
+pkgrel=0
+pkgdesc="Idris - A Language with Dependent Types"
+url="http://www.idris-lang.org"
+arch="x86_64 armhf"
+license="bsd3"
+depends="libffi gmp ncurses libffi-dev gmp-dev"
+makedepends="bash ghc ghc-dev cabal libffi gmp libffi-dev gmp-dev ncurses-dev zlib-dev"
+install=""
+subpackages="$pkgname-doc"
+source="$pkgname-$pkgver.tar.gz::https://github.com/idris-lang/Idris-dev/archive/v$pkgver.tar.gz"
+builddir="$srcdir/Idris-dev-$pkgver"
+
+prepare() {
+	cd "$builddir"
+	(
+		export HOME="$builddir/deps"
+		cabal update || return 1
+		cabal install libffi -v || return 1
+		cabal install --dependencies-only -v || return 1
+	) || return 1
```

Comment:
It’s not needed to spread `|| return 1` across abuilds anymore (yey!), abuild finally runs with `set -e`. :)